src/SwiftUI/SwiftUI/SwiftUIViewExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+using Swift;
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	public unsafe static class SwiftUIExtensions
```

Comment:
```suggestion
	public unsafe static class ViewExtensions
```

Reply:
I'm going to rename the file to match the class. I'm not a fan of them being out of sync.